vendor/k8s.io/kubernetes/staging/src/k8s.io/apimachinery/pkg/util/intstr/intstr.go

```diff
@@ -143,6 +143,9 @@ func (intstr *IntOrString) Fuzz(c fuzz.Continue) {
 }
 
 func GetValueFromIntOrPercent(intOrPercent *IntOrString, total int, roundUp bool) (int, error) {
+	if intOrPercent == nil {
```

Comment:
per my comment upstream.  I think we need to avoid value compression.  You can have a defaulttozeroifnil function that you wrap the call to this with, but this method needs the distinction.